behind-the-veil-siteroot/server/main.js

```diff
@@ -45,7 +45,9 @@ Meteor.startup(async () => {
 
     if (Meteor.isServer) {
         // process.env.MAIL_URL = mailUrl;
-        Accounts.emailTemplates.from = fromUser;
+        // Accounts.emailTemplates.from = fromUser;
```

Comment:
do you reckon we need this anymore then?

Reply:
nah good point ill get rid of it